adapt/engine.py

```diff
@@ -368,3 +434,41 @@ def register_intent_parser(self, intent_parser, domain=0):
             self.register_domain(domain=domain)
         self.domains[domain].register_intent_parser(
             intent_parser=intent_parser)
+
+    def drop_intent_parser(self, parser_name, domain):
+        """Drop a registered intent parser.
+
+        Arguments:
+            parser_name (str): parser name to drop.
+            domain (str): domain to drop from
+
+        Returns:
+            (bool) True if an intent parser was dropped else false.
+        """
+        return self.domains[domain].drop_intent_parser(parser_name)
+
+    def drop_entity(self, domain, **kwarg):
+        """Drop all entities mathching the given query or match function
+
+        Arguments:
```

Comment:
The docstring here does not match the method signature. It seems a bit messy to pass through the `**kwargs` instead of enumerating the actual valid values.

Reply:
Thought it would be more stable to just use the **kwargs but I've updated as you suggest.